google/ggrc-core
# GGRC-3675 Add tip for Network Zone

# Issue description

Network Zone" column doesn't have a hint in csv template for Process and System

# Steps to test the changes

1.Log into GGRC app.
2.Goto Export page.
3.Select the Object Type  and export the csv file.
4.Open the excel and check for the Network Zone column , here you can see the tip message
"Allowed values are:
Service
Core
3rd Party
Prod
Corp"

# Solution description

Added variable *NZ_OPTIONS* and updated \_aliases in file src/ggrc/models/mixins/__init__.py 

# Sanity checklist




# PR Review checklist
